mino/minoch/mod.go

```diff
@@ -29,7 +43,7 @@ type Minoch struct {
 }
 
 // NewMinoch creates a new instance of a local Mino instance.
-func NewMinoch(manager *Manager, identifier string) (*Minoch, error) {
+func NewMinoch(manager *Manager, identifier string) *Minoch {
```

Comment:
to avoid later trouble, maybe that would be better to have a `minoch.MustCreate()` function that panics, and let this function return an error for the cases we want to be safe ? (I hear you saying "minoch is for testing purposes anyway" 😄)

Reply:
No that's actually a good compromise. It's just that it's annoying to have to handle those kind of errors (like the namespace and the rpc) that can happen only during instantiation if you're really doing something wrong. In that case I think it's fair to take a panic in the face.

Reply:
What do you think about a mino.MustCreateRPC(name string, ...) mino.RPC function and similarly for the namespace ?

Reply:
mmm `MakeNamespace` can easily return an error with a malformed namespace, so I'm not convinced. `MakeRPC`, on the other hand, has no reason to return an error, at least with minogrpc, so `MustCreateRPC` seems like a good idea.

Reply:
For the record, we've decided to do as @nkcr said for Minoch and change the namespace and rpc functions to panic if necessary.